### ISSUE TITLE
MythTV 0.28 has renamed the 'Images' Storage Group 'Photographs'

### ISSUE THREAD

**Issue by billmeek** [MythTV-Clients/MythTV-Android-Frontend#204]
MythTV commit: http://code.mythtv.org/cgit/mythtv/commit/?id=1b62cb4

References #174 (this is a non-translatable string in MythTV, so I
can change the German strings.xml.) This is the 3rd name for this
Storage Group, should we keep trying to change a moving target?

**Comment by billmeek** [comment]
Might want to hold off on any changes, from the MythTV developers IRC:

```
Captain_Murdoch:    stuartm, I don't think it was mentioned, but perhaps 'Gallery' instead of Photographs,
```

This is where we started.

**Comment by dmfrey** [comment]
This can be closed as I implemented this when I last pushed a fix to #194
